### PR TITLE
Remove unused columns from users table.

### LINKED ISSUE #264: Consider removing "users.csr" and "users.crt" db columns
Seems that app uses "certificates" table instead

### PR DESCRIPTION
**Pull request by teadur**
In production there is no data in these column, nothing to test.
Closes #264

### CODE REVIEW

File: db/migrate/20191008024334_remove_unused_from_users.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class RemoveUnusedFromUsers < ActiveRecord::Migration
```
Comment (artur-beljajev) [inline db/migrate/20191008024334_remove_unused_from_users.rb]: Might be good idea to name it `RemoveUnusedColumnsFromUsers`. Or to follow existing convention: `RemoveColumn1Column2FromTableName`.

Comment (teadur) [reply db/migrate/20191008024334_remove_unused_from_users.rb]: Should be better now.

**Comment by teadur** [comment]
I think it makes sense to squash and merge either way, so whoever pushes it to master can do it.

**Comment by artur-beljajev** [comment]
@teadur

I have squashed them into single one. Also edited commit message. https://chris.beams.io/posts/git-commit/ might be a good read on best practices.

Make sure `structure.sql` does not have unwanted changes. Newline at the end was missing.

**Pull request merged**